projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py

```diff
@@ -154,13 +155,18 @@ def update(
             deployment.job_version = job_version
         if vdk_version:
             deployment.vdk_version = vdk_version
+        deployment.enabled = enabled
+
         if job_version:
             self.__update_job_version(name, team, deployment, output)
-        elif vdk_version:
+        elif vdk_version or enabled is not None:
             self.__update_deployment(name, team, deployment)
-            log.info(
-                f"Deployment of Data Job {name} updated to use vdk version {vdk_version}."
-            )
+            msg = f"Deployment of Data Job {name} updated: "
```

Comment:
This message seems a bit confusing. Maybe could be re-written to something like, `Data Job deployment updated; Status: `?

Reply:
```
Deployment of Data Job xxx updated: vdk version: 1.3; enabled; 
```

Reply:
For consistency's sake, maybe replace the `:` after 'updated' with a `;`, and add 'status: ' before 'enabled'.

Also, `nit:` on line 168 below, there is NO `;` after "enabled", while there is one after "disabled".